clients/include/flops.hpp

```diff
@@ -262,6 +262,25 @@ constexpr double ger_gflop_count(rocblas_int m, rocblas_int n)
     return (2.0 * m * n + min) / 1e9;
 }
 
+/* \brief floating point counts of SPR */
+template <typename T>
+constexpr double spr_gflop_count(rocblas_int n)
+{
+    return (n * (n + 1) + n) / 1e9;
```

Comment:
Should be
```
return (double(n) * (n + 1) + n) / 1e9;
```

i.e., convert to `double` before doing calculations. Only one conversion at the innermost level is needed; the rest are implicit.